hack/run-e2e.sh

```diff
@@ -25,7 +25,8 @@ export KUBE_TEST_REPO_LIST=${WORKSPACE}/repo-list.yaml
 
 # Download the list of tests to exclude.
 curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/exclude_conformance_test.txt -o ${WORKSPACE}/exclude_conformance_test.txt
-EXCLUDED_TESTS=$(cat ${WORKSPACE}/exclude_conformance_test.txt |
+# Ignore lines starting with "#"
+EXCLUDED_TESTS=$(grep -v ^# ${WORKSPACE}/exclude_conformance_test.txt |
```

Comment:
Did you test this? Does the # need to be escaped or quoted?

Reply:
Yep, it works. Added quotes and re-pushed anyway :)